lib/view/home_page.dart

```diff
@@ -59,11 +60,30 @@ class HomePageState extends State<HomePage> {
     super.initState();
   }
 
+  List<String> getSuggestion() {
+    var suggestion = <String>[];
+    for (var person in allPersons) {
+      if (suggestion.contains(person.name) == false) {
```

Comment:
modify 
if(suggestion.contains(person.name)== false)   
to 
if(!suggestion.contains(person.name))